README.md

```diff
@@ -1,3 +1,183 @@
 # hms-build-chart-workflows
 
-Workflows to build HMS Helm charts.
\ No newline at end of file
+- [hms-build-chart-workflows](#hms-build-chart-workflows)
+  - [Build and Release charts workflow](#build-and-release-charts-workflow)
+    - [Workflow inputs](#workflow-inputs)
+    - [Workflow secrets](#workflow-secrets)
+    - [Example usage](#example-usage)
+  - [Charts lint, test, and scan workflow](#charts-lint-test-and-scan-workflow)
+    - [Workflow inputs](#workflow-inputs-1)
+    - [Workflow secrets](#workflow-secrets-1)
+    - [Lint and Test job](#lint-and-test-job)
+    - [Scan image job](#scan-image-job)
+    - [Scan chart job](#scan-chart-job)
+    - [Example usage](#example-usage-1)
+  - [Release model](#release-model)
+
+The hms-build-image-workflows repository contains Github Action workflows to facilitate the process of building and testing Helm charts from HMS repositories.
+
+## Build and Release charts workflow
+
+![](docs/build_and_release_charts.svg)
+
+The build and release charts workflow is composed of a single job that is used to package and publish Helm charts from a HMS Helm chart repository. Stable artifacts are produced when a PR or branch is merged to main. For all other pushes or PRs an unstable artifact is produced.
+
+At a high level this workflow performs the following actions:
+1. Build and package Helm charts. For non-main only changed charts (when compared to the main branch) are build. When on workflow run is triggered on the main branch to create a stable Helm chart, only the charts that have not been previously released will be built. 
```

Comment:
This could use a little work. (however what do I know, maybe I should just improve my reading skills)
In the second sentence change build to built. In the third change "When  on" to "When the". Or rework the text. Here is what I understand as being the way it works:
1. Build and Package Helm charts.
    1. Changes to the main branch will publish stable builds of the Helm charts that have not previously been released.
     2. Changes to any other branch will publish unstable builds for the Helm charts that changed.

Reply:
I like your wording better.